subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfiguration.java

```diff
@@ -119,7 +119,7 @@
     private final CompositeDomainObjectSet<PublishArtifact> inheritedArtifacts;
     private final DefaultPublishArtifactSet allArtifacts;
     private final ConfigurationResolvableDependencies resolvableDependencies = new ConfigurationResolvableDependencies();
-    private final ListenerBroadcast<DependencyResolutionListener> dependencyResolutionListeners;
+    private ListenerBroadcast<DependencyResolutionListener> dependencyResolutionListeners;
```

Comment:
I don't see a reason why the field should not be `final`. Can we change that back?